e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,22 @@
+import { AppPage } from './app.po';
```

Comment:
at the moment these are just generated tests. Blow them away if they don't serve a purpose